docs/core/install/linux-debian.md

```diff
@@ -94,6 +94,8 @@ sudo apt-get update
 sudo apt-get upgrade
 ```
 
+If you've upgraded your distribution since installing .NET, the package feed may be out of date. Run the installation instructions for your particular version to upgrade to the appropriate package feeds for .NET.
```

Comment:
> the package feed may be out of date.

When I'm thinking of apt, "package feed" means the feed data to me, so `apt update` would fix this. Maybe something like:

> If you've upgraded your Linux distribution since installing .NET, you may need to reconfigure the Microsoft package repository. Run the installation instructions for your current distribution version to upgrade to the appropriate package repository for .NET updates.

/cc @NikolaMilosavljevic @MichaelSimons @dleeapho